lib/views/video_view/video_player.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/views/video_view/chat_video_view.dart';
+import 'package:gocast_mobile/views/video_view/video_player_controller.dart';
+
+class VideoPlayerPage extends StatefulWidget {
+  final String videoSource;
+  final VideoSourceType sourceType;
+  final String title;
```

Comment:
I'd suggest thinking of replacing these with a `final Stream stream;` so that whenever you create a new VideoPlayerPage, you only have to pass the Stream as an argument without having to pass each field separately or needing to keep a state of the current stream.

Reply:
This will be done in this [issue](https://github.com/orgs/TUM-Dev/projects/14/views/6?sliceBy%5Bvalue%5D=GravityDarkLab&pane=issue&itemId=47620462)

Reply:
This will be done in this [issue](https://github.com/orgs/TUM-Dev/projects/14/views/6?sliceBy%5Bvalue%5D=GravityDarkLab&pane=issue&itemId=47620462)
As this PR is mainly about the UI.

Reply:
Perfect 👍